README.md

```diff
@@ -79,3 +83,51 @@ Go to the PISP demo app and try to send money to a payee. You should see that th
 # Configurations
 
 Take a look at [src/lib/config](https://github.com/mojaloop/pisp-demo-server/blob/master/src/lib/config.ts) to see all the different aspects of the server that you can configure.
+
+# Mock Data for Account Linking
+
+## Mocking Available Financial Service Providers 
+
+Follow these steps to display a list of available Financial Service Providers on the account linking tab.
+
+1. Create a new collection called `participants`
+2. Inside the collection `participants`, add as many different FSP documents as you want.
+
+An FSP document consists of a Map that looks like so:
+
+```
+{
+  fspId: "hsbc",
+  name: "HSBC Bank"
+}
+```
+
+When the server is fully functional, the server will periodically update this list and populate the 'participants' collection so this won't be needed anymore.
+
+## Mocking Accounts for Associated Accounts
+
+1. In the Consent object, that you are working with, add an `accounts` field of type 'array'.
+2. In that array, add as many Account documents as you want.
+
+An Account document consists of a Map that looks like so:
+
+```
+{
+  id: "fspId",
```

Comment:
`Account.id` suggests that `id` is of an `Account` but it is FSP.id -> I suggest to use `fspId` instead

Reply:
or maybe in this example, `fspId` or `fspA` means Account's Id allocated/handled by given FSP -> so better is to use more descriptive values in this example...

Reply:
That makes sense @eoln 
I will change the value.
The property names are kind of set already, I think I mistakenly wrote fspId, the id should actually be the id of the account.

Reply:
I changed them to be similar to those in the linking diagrams. What do you think?